templates/navBar.php

```diff
@@ -32,7 +32,7 @@
   }  
 .navbar-brand:hover
 {
-  color: red;
+  color: #fff;
```

Comment:
why a new color is added? This can be removed right?

Reply:
Actually there is a by default css for a tag.
On Hover on a tag its colour change to Blue.
Its by default so i have to overwrite it
@siv2r

Reply:
Okay